webknossos/webknossos/dataset/dataset.py

```diff
@@ -1958,3 +1963,12 @@ def allowed_teams(self, allowed_teams: Sequence[Union[str, "Team"]]) -> None:
             assert (
                 dataset_update_teams_response.status_code == 200
             ), dataset_update_teams_response
+
+    @property
+    def folder(self) -> RemoteFolder:
+        return RemoteFolder.get_by_id(self._get_dataset_info().folder_id)
+
+    @folder.setter
+    def folder(self, folder: RemoteFolder) -> None:
+        """Move the dataset to a folder. Specify the folder like this `RemoteFolder.get_by_path("Root/Folder_A")`."""
```

Comment:
I think the root is called `Datasets` by default?

```suggestion
        """Move the dataset to a folder. Specify the folder like this `RemoteFolder.get_by_path("Datasets/Folder_A")`."""
```